src/components/sessions/activeSession/SessionProgress.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Icon } from 'semantic-ui-react'
+import { FormattedMessage } from 'react-intl'
+
+import Question from './Question'
+
+const SessionProgress = ({ intl }) => (
+  <div className="container">
+    <div className="sessionContainer">
+      <div className="topRow">
+        <div><Icon name="time" /> <FormattedMessage
+          defaultMessage="Start"
+          id="pages.runningSession.sessionProgress.paragraph.start" // TODO correct naming of identifier
+        /></div>
+        <div><Icon name="play circle" /> <FormattedMessage
```

Comment:
@pascalzehnder Still needs to be done.. I have to manually reformat the files you edit..